src/Freya.Core/Integration.fs

```diff
@@ -18,12 +18,12 @@
 //
 //----------------------------------------------------------------------------
 
-[<AutoOpen>]
-module Freya.Core.Integration
+module Freya.Integration
```

Comment:
I think those, but with [<AutoOpen>], so people only need to open
Freya.Core to get integration and pipelining available fully. I'll merge
locally in to my branch shortly and send an updated PR through soon!

On 28 April 2015 at 17:19, Ryan Riley <notifications@github.com> wrote:

> In src/Freya.Core/Integration.fs
> <https://github.com/freya-fs/freya/pull/89#discussion_r29259759>:
>
> > @@ -18,12 +18,12 @@
> >  //
> >  //----------------------------------------------------------------------------
> >
> > -[<AutoOpen>]
> > -module Freya.Core.Integration
> > +module Freya.Integration
>
> Sounds good to me. I was trying to make minimal changes. Pipeline and
> Integration are slightly different from the Core module types, so I don't
> think it's bad to leave them, but I also can't argue against using the same
> module/namespace for all.
>
> Are you thinking Freya.Core.Pipeline and Freya.Core.Integration or just
> Freya.Core for everything?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freya-fs/freya/pull/89/files#r29259759>.
>